pages/about.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Olive Lavine</title>
+  <link rel="stylesheet" href="/stylesheets/style.css">
 </head>
 <body>
-  
+  <header>
+    <img src="/images/olive.jpg" alt="A black and white photo of a person 
+    from the shoulders up, with pale skin and straight long hair in a graduation 
+    cap and gown, against a background of clear skies and a tall city building" />
+    <h1>Olive Henzel Lavine</h1>
+    <nav>
+      <menu>
```

Comment:
Nice choice. I saw your comment about `menu` support in the comprehension questions, were there issues you ran into, or was it documentation that brought that up?